spec/presenters/summary/html_presenter_spec.rb

```diff
@@ -18,20 +18,42 @@
       ).to receive(:show?).and_return(true)
     end
 
-    it 'has the right sections in the right order' do
```

Comment:
HtmlPresenter doesn't care about conditionality. It is a smoke test so to speak, to ensure all the sections appear in the right order. We allow `#show?` on purpose to always return true on the BaseSection to accomplish this.

Conditionality is tested at the section level, in each of the specific sections.